main/2024/CG-10-08.md

```diff
@@ -23,10 +23,85 @@ No registration is required for VC meetings. The meeting is open to CG members o
 
 ## Meeting Notes
 
-To be filled in after the meeting.
-
 ### Attendees
 
+ - Hao Tran
+ - Derek Schuff
+ - Paul Dennis
+ - Zalim Bashorov
+ - Jamey Sharp
+ - David Thompson
+ - Aruokhai Joshua
+ - Jeff Charles
+ - Hunter Demeyer
+ - Saul Cabrera
+ - Alex Crichton
+ - Luke Wagner
+ - Yury Delendik
+ - Paolo Severini
+ - Nick Fitzgerald
+ - Manos Koukoutos
+ - Andrew Brown
+ - Nuno Pereira
+ - Chris Fallin
+ - Francis McCabe
+ - Julien Pages
+ - Adam Bratschi-Kaye
+ - Brendan Dahl
+ - Jake Enget
+ - Emanuel Ziegler
+ - Jakob Kummerow
+ - Oscar Spencer
+ - Heejin Ahn
+ - Mattias Liedtke
+ - Sam Clegg
+ - Ryan Hunt
+ - Bailey Hayes
+ - Ilya Rezvov
+ - Daniel Lehmann
+ - Chris Woods
+
+
+
 ### Proposals and discussions
 
+1. Update on and (possible) phase 2 vote for [128-bit-arithmetic](https://github.com/WebAssembly/128-bit-arithmetic) (Alex Crichton) 
```

Comment:
@alexcrichton link to slides?

Reply:
Indeed! 

https://docs.google.com/presentation/d/1K8Dq7EHfUDuaGmA4JiiltNCHEgNf8S5i0cs5sKcdjsQ/edit?usp=sharing